src/main/scala/viper/gobra/ast/internal/Program.scala

```diff
@@ -109,6 +111,7 @@ case class Method(
                  override val results: Vector[Parameter.Out],
                  override val pres: Vector[Assertion],
                  override val posts: Vector[Assertion],
+                 override val terminationMeasure: Option[Vector[Assertion]],
```

Comment:
On a second thought, I would change the type of terminationMeasure to `Vector[DecreaseClause]`. See my comments below to see what `DecreaseClause` stands for.